tests/cl-genesis-positions/get_data.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sort"
+)
+
+func GetUniV3SubgraphData(pathToSaveAt string) {
+	// Set the subgraph URL and query.
+	const (
+		subgraphURL = "https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3"
+		query       = `{
+			positions(
+				first: 1000
```

Comment:
asking to learn: it seems that this makes it return batches of 1000, what is it that we are batching? The Positions or the ticks?

Reply:
The positions. 1000 is the max number it is possible to get per query